src/Stratis.Bitcoin/Utilities/MemoryCache.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Stratis.Bitcoin.Utilities
+{
+    /// <summary>
+    /// Memory cache that implements the Least Recently Used (LRU) standard.
```

Comment:
lru is not a standard - https://en.wikipedia.org/wiki/Cache_replacement_policies

Reply:
Fixed
Replaced word 'standard' with 'policy'